modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,216 @@
++
+[source, bash]
+----
+moactl completion > /etc/bash_completion.d/moactl
+source /etc/bash_completion.d/moactl
+----
++
+. Run the following command to verify that your AWS account has the necessary permissions:
++
+[source,bash]
+----
+$ moactl verify permissions
+I: Validating SCP policies...
+I: AWS SCP policies ok
+----
++
+.. Verify that your AWS account has the necessary quota to deploy an OpenShift cluster. Sometimes quota varies by region, which may prevent you from deploying:
++
+[source,bash]
+----
+$ export AWS_DEFAULT_REGION=us-west-2 && moactl verify quota
+I: Validating AWS quota...
+E: Insufficient AWS quotas
+E: Service ebs quota code L-FD252861 Provisioned IOPS SSD (io1) volume storage not valid
+----
++
+[NOTE]
+====
+Sometimes quota varies by region, which may prevent you from deploying. If you receive any errors, try a different region.
+====
++
+If you need to increase your quota, navigate to your link:https://aws.amazon.com/console/[AWS console], and request a quota increase for the service that failed.
++
+Once both the permissions and quota checks pass, proceed to initializing your AWS account.
++
+. Initializing your AWS account
++
+In this step you log in to your Red Hat account using `moactl`, and then initialize your AWS account.
++
+.. Log in to your Red Hat account with `moactl`.
++
+[NOTE]
+====
+If you do not already have a Red Hat account, [create one here](https://cloud.redhat.com/). Be sure to accept the required terms and conditions. Then, check your email for a verification link.  
+
+After creating your Red Hat account, follow this link to [get an offline access token](https://cloud.redhat.com/openshift/token/moa
+).
+====
++
+.. Run the following command to log in to your Red Hat account with moactl. Replace `<my-offline-access-token>` with your token:
++
+[source, bash]
+----
+$ moactl login --token="<my-offline-access-token>"
+----
++
+. Verify `moactl` login and aws-cli defaults
++
+.. Run the following command to verify your Red Hat and AWS credentials are setup correctly.  Check that your AWS Account ID, Default Region and ARN match what you expect.  You can safely ignore the rows beginning with OCM for now (OCM stands for OpenShift Cluster Manager).
++
+[source, bash]
+----
+$ moactl whoami
+AWS Account ID:               000000000000
+AWS Default Region:           us-east-2
+AWS ARN:                      arn:aws:iam::000000000000:user/hello
+OCM API:                      https://api.openshift.com
+OCM Account ID:               1DzGIdIhqEWyt8UUXQhSoWaaaaa
+OCM Account Name:             Your Name
+OCM Account Username:         you@domain.com
+OCM Account Email:            you@domain.com
+OCM Organization ID:          1HopHfA2hcmhup5gCr2uH5aaaaa
+OCM Organization Name:        Red Hat
+OCM Organization External ID: 0000000
+----
++
+. Initialize your AWS account
```

Comment:
Step 5 and 7 are both initializing the AWS account. Are there two ways to do it? Or can they be combined?

Reply:
I updated 5 to be "Prepare your AWS account for cluster deployment." I made a number of other small changes: the steps in this section should all be under this heading, so I updated the nesting.